content/en/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md

```diff
@@ -59,12 +59,10 @@ kubectl label --dry-run=server --overwrite ns --all \
 ### Applying to all namespaces
 
 If you're just getting started with the Pod Security Standards, a suitable first step would be to
-configure all namespaces as `privileged` but set up audit annotations for a stricter level such as
-`baseline`:
+configure all namespaces with audit annotations for a stricter level such as `baseline`:
```

Comment:
is it worth calling out that no enforce level is being explicitly selected, and why?